include/pmacc/cuSTL/container/copier/Memcopy.hpp

```diff
@@ -48,9 +48,9 @@ struct Memcopy<1>
                     const Type* source, const math::Size_t<0>, const math::Size_t<1>& size,
                     flags::Memcopy::Direction direction)
     {
-            const cudaMemcpyKind kind[] = {cudaMemcpyHostToDevice, cudaMemcpyDeviceToHost,
-                                     cudaMemcpyHostToHost, cudaMemcpyDeviceToDevice};
-            CUDA_CHECK(cudaMemcpy(dest, source, sizeof(Type) * size.x(), kind[direction]));
+            const cuplaMemcpyKind kind[] = {cuplaMemcpyHostToDevice, cuplaMemcpyDeviceToHost,
+                                     cuplaMemcpyHostToHost, cuplaMemcpyDeviceToDevice};
+            CUDA_CHECK(cuplaMemcpy(dest, source, sizeof(Type) * size.x(), kind[direction]));
```

Comment:
Should this still be `CUDA_CHECK` ? (same in other places)

Reply:
changed to `CUPLA_CHECK`